data/Glitched World/Overworld.json

```diff
@@ -24,6 +25,14 @@
             "Desert Colossus": "can_play(Requiem_of_Spirit) and can_leave_forest"
         }
     },
+    {
+        "region_name": "Egg Hatch",
+        "hint": "Link's Pocket",
+        "locations": {
+            "Hatch Weird Egg": "is_child and (Weird_Egg or skip_child_zelda)",
```

Comment:
These technically require time passing, though, right? Weird Egg is easy enough because it's used in a place where time passes. (That makes me question whether it's worthwhile to have this split for Weird Egg.) Pocket Egg, not so much--mostly it'll rely on the ER guarantee to a time-passing region.